understanding/target-size-minimum.html

```diff
@@ -53,330 +53,238 @@
-                        include sliders, color pickers displaying a gradient of colors, or editable areas
-                        where you position the cursor.
+                     <p class="note-title marker">注記</p>
+                     <p>ターゲット内の位置に基づいて空間的に値を選択できるターゲットは、この達成基準の目的上、一つのターゲットとみなされる。例としては、スライダー、色のグラデーションを表示するカラーピッカー、カーソルを置いた位置を編集できる領域などがある。
                      </p>
                   </div>
                   <div class="note">
-                     <p class="note-title marker">Note</p>
-                     <p>For inline targets the line-height should be interpreted as perpendicular to the flow
-                        of text. For example, in a language displayed vertically, the line-height would be
-                        horizontal.
+                     <p class="note-title marker">注記 </p>
+                     <p>インラインのターゲットの場合、行の高さはテキストのフローに対して垂直方向として解釈されるべきである。例えば、縦書きで表示される言語では、行の高さは水平方向となる。
                      </p>
                   </div>
                </div>
             </aside>
             <div class="excol-all"></div>
             <section id="brief">
-               <h2>In Brief</h2>
+               <h2>要約</h2>
                <dl>
                   
-                  <dt>Goal</dt>
-                  <dd>Make controls easier to activate.</dd>
+                  <dt>目標</dt>
+                  <dd>コントロールを作動しやすくする。</dd>
                   
-                  <dt>What to do</dt>
-                  <dd>Ensure targets meet a minimum size or have sufficient spacing around them.</dd>
+                  <dt>何をすればよいか</dt>
+                  <dd>ターゲットが最低限のサイズを満たしている、又はターゲットの周囲に十分な間隔があることを確認する。</dd>
                   
-                  <dt>Why it's important</dt>
-                  <dd>Some people with physical impairments cannot click small buttons that are close together.</dd>
+                  <dt>なぜそれが重要か</dt>
+                  <dd>身体障害のある人の中には、近くにある小さなボタンをクリックできない人もいる。</dd>
                   
                </dl>
             </section>
             <section id="intent">
-               <h2>Intent</h2>
-               <p>The intent of this Success Criterion is to help ensure targets can be easily activated
-                  without accidentally activating an adjacent target.  Users with dexterity limitations
-                  and those who have difficulty with fine motor movement find it difficult to accurately
-                  activate small targets when there are other targets that are too close.  Providing
-                  sufficient size, or sufficient spacing between targets, will reduce the likelihood
-                  of accidentally activating the wrong control.
+               <h2>意図</h2>
+               <p>この達成基準の目的は、隣接するターゲットを誤って起動することなく、ターゲットを容易に起動できるようにするのを保証することである。巧緻性に限界がある利用者、及び細かい運動が困難な利用者は、別のターゲットが近すぎると、小さなターゲットを正確に起動することが困難になる。十分なサイズ、又はターゲット間に十分な間隔を設けることで、間違ったコントロールが誤って起動なる可能性が低くなる。
```

Comment:
activate の訳が、「起動」「作動」で揺れていたので、「作動」で統一してみましょうか。

```suggestion
               <p>この達成基準の目的は、隣接するターゲットを誤って作動させることなく、ターゲットを容易に作動できるようにするのを保証することである。巧緻性に限界がある利用者、及び細かい運動が困難な利用者は、別のターゲットが近すぎると、小さなターゲットを正確に作動させることが困難になる。十分なサイズ、又はターゲット間に十分な間隔を設けることで、間違ったコントロールが誤って作動する可能性が低くなる。
```